### ISSUE TITLE
Request for bottom toolbar to dissappear from view or be transparent

### ISSUE THREAD

**Issue by emclaren** [mozilla/hubs#3468]
Several people have mentioned when touring the new UI on desktop that they would appreciate the following to improve immersion in the space:

Either 

1) Having an option for the bottom toolbar to automatically drop out of view when the cursor isn't near it
or
2) Having an option to change the alpha value for the bottom toolbar background so you can see whats in the room behind it

**Comment by robertlong** [comment]
This is interesting. I get the reasoning, but I think I'm personally against it.

On macOS and my Linux machine I use the behavior described in 1 for my dock. By default the dock is quite large and I like to free up the screen real estate. However if it isn't implemented well that hide/show animation can be really annoying. It sometimes is on my Linux machine. This would also be somewhat complex for popover menus. I think this would be a lot of work to do correctly and possibly not a great experience for users.

I prefer the second option if we were to implement this as a preference. However, I'm not rendering anything behind the toolbar right now. We actually gain some performance by not rendering this pixels, not that it's that huge of a deal, but making it transparent will lose those gains.

Overall, I'd like to sit on this design for a while and get feedback.

**Comment by emclaren** [comment]
I'm adding a label of P3 because this seems like something we could take a closer look at once the new UI has launched.

**Comment by mcroud** [comment]
Do we know if those requesting this feature are aware of the existing hide UI `\` button ?

**Comment by misslivirose** [comment]
@mcroud I don't think that we have a sense of whether or not this ask is in addition to the hide UI feature or because the 'hide UI' may not be discoverable. I would probably lean towards making it easier to know you can hide the UI with a toggle over exploring ways to auto-hide the UI.
